workshop/controller-workshop.json

```diff
@@ -6,167 +6,115 @@
-        "pip-redis",
-        "pip-jsonschema",
-        "pip-ntfy",
-        "buildah",
-        "podman",
-        "skopeo",
+        "distro-perl-tools",
+        "distro-perl-modules",
+        "container-tools",
         "container-storage-fuse",
         "httpd",
         "elasticsearch_from_es",
         "elasticsearch_post",
         "java",
         "nodejs",
-        "motd",
-        "bc",
-        "sshpass",
-        "dnf",
+        "non-distro-node-deps",
         "gh-cli-repo",
         "gh-cli"
       ]
     }
   ],
- "requirements": [
-     {
-        "name": "pip-redis",
-        "type": "python3",
-        "python3_info": {
-            "packages": [
-                "redis",
-                "hiredis"
-            ]
-        }
-     },
-    {
-      "name": "openssl-devel",
-      "type": "distro",
-      "distro_info": {
-        "packages": [
-          "openssl-devel"
-        ]
-      }
-    },
-    {
-      "name": "zlib-devel",
-      "type": "distro",
-      "distro_info": {
-        "packages": [
-          "zlib-devel"
-        ]
-      }
-    },
-    {
-      "name": "bind-utils",
-      "type": "distro",
-      "distro_info": {
-        "packages": [
-          "bind-utils"
-        ]
-      }
-    },
-    {
-      "name": "iproute",
-      "type": "distro",
-      "distro_info": {
-        "packages": [
-          "iproute"
-        ]
-      }
-    },
+  "requirements": [
     {
-      "name": "xz",
+      "name": "core-utils",
       "type": "distro",
       "distro_info": {
         "packages": [
+          "curl",
+          "tar",
+          "cpio",
           "xz",
-          "xz-devel"
-        ]
-      }
-    },
-    {
-      "name": "hostname",
-      "type": "distro",
-      "distro_info": {
-        "packages": [
-          "hostname"
-        ]
-      }
-    },
-    {
-      "name": "jq",
-      "type": "distro",
-      "distro_info": {
-        "packages": [
-          "jq"
+          "gzip",
+          "jq",
+          "git",
+          "cpio",
+          "findutils",
+          "hostname",
+          "iputils",
+          "iproute",
+          "bind-utils",
+          "procps-ng",
+          "git",
+          "bc",
+          "sshpass"
         ]
       }
     },
     {
-      "name": "procps-ng",
+      "name": "python39",
       "type": "distro",
       "distro_info": {
         "packages": [
-          "procps-ng"
+          "python39",
+          "python3-pip"
         ]
       }
     },
     {
-      "name": "gcc",
+      "name": "distro-python-deps",
       "type": "distro",
       "distro_info": {
         "packages": [
-          "gcc"
+          "python3-numpy",
+          "python3-pandas",
+          "python3-pillow"
         ]
       }
     },
     {
-      "name": "make",
-      "type": "distro",
-      "distro_info": {
+      "name": "non-distro-python-deps",
+      "type": "python3",
+      "python3_info": {
         "packages": [
-          "make"
+          "Keras",
+          "Pillow",
```

Comment:
@mrbojangles3 are these dependencies that you are using for something?

Reply:
Nope. Those are some ML/AI things. not sure how those snuck in there.